### PR TITLE
react-mapexplorer: move package to indigo-js

### PR DESCRIPTION
**Pull request by such**
Used https://github.com/transitive-bullshit/react-modern-library-boilerplate to generate the project.

### CODE REVIEW

File: packages/react-mapexplorer/README.template.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    return (
+      <MyComponent />
+    )
+  }
+}
+```
+
+## License
+
+MIT © [Travis Fischer](https://github.com/transitive-bullshit)
```
Comment (t-bast) [inline packages/react-mapexplorer/README.template.md]: Is this file really needed? The example doesn't really use our mapexplorer component, does it?

File: packages/react-mapexplorer/example/README.md [hunk 2]
```diff
@@ -0,0 +1,2164 @@
...
+Alternatively you may use `yarn`:
+
+```sh
+yarn add jest-enzyme
+```
+
+Import it in [`src/setupTests.js`](#initializing-test-environment) to make its matchers available in every test:
+
+```js
+import 'je
```
Comment (t-bast) [inline packages/react-mapexplorer/example/README.md]: Same, do we want to keep this?

File: packages/react-mapexplorer/example/package.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "react-mapexplorer-example",
+  "homepage": "https://stratumn.github.io/react-mapexplorer",
+  "version": "0.0.0",
+  "private": true,
+  "license": "MIT",
```
Comment (t-bast) [inline packages/react-mapexplorer/example/package.json]: Should we update this to Apache 2.0?

File: packages/react-mapexplorer/example/package.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  "dependencies": {
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1",
+    "react-scripts": "1.0.13"
```
Comment (t-bast) [inline packages/react-mapexplorer/example/package.json]: nit: react-scripts can be moved to dev-dependencies

File: packages/react-mapexplorer/README.template.md (hunk 1 above)
Comment (such) [reply packages/react-mapexplorer/README.template.md]: not needed indeed!

File: packages/react-mapexplorer/example/README.md (hunk 2 above)
Comment (such) [reply packages/react-mapexplorer/example/README.md]: not really

File: packages/react-mapexplorer/example/package.json (hunk 3 above)
Comment (such) [reply packages/react-mapexplorer/example/package.json]: yeah it's not a big deal since we want to get rid of example as soon as agent-ui uses react-mapexplorer, but I'll change it in the meantime.

File: packages/react-mapexplorer/example/package.json (hunk 4 above)
Comment (such) [reply packages/react-mapexplorer/example/package.json]: ok!

File: packages/react-mapexplorer/example/package.json (hunk 3 above)
Comment (t-bast) [reply packages/react-mapexplorer/example/package.json]: Ok makes sense, no problem if you don't do it in this PR
